vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/document/TextDocumentEditor.java

```diff
@@ -176,7 +176,6 @@ private static String positionalConcat(String line, Position startPos, Position
 
     private static List<String> split(String text)
     {
-        return Stream.of(splitByWholeSeparatorPreserveAllTokens(text, System.lineSeparator()))
-                     .collect(Collectors.toList());
+        return Splitter.on(LINE_SEPARATOR_PATTERN).splitToStream(text).collect(Collectors.toList());
```

Comment:
why not `String#split`?

Reply:
it was initially done with splitByWholeSeparatorPreserveAllTokens to preserve all tokens, just splilt does not preserve tokens

Reply:
`java.lang.String#split(java.lang.String, int)` with limit = -1 ?

Reply:
fixed for now https://github.com/vividus-framework/vividus-studio/issues/327 :)